python_modules/dagster/dagster/_core/definitions/asset_check_result.py

```diff
@@ -53,3 +77,59 @@ def __new__(
             success=check.bool_param(success, "success"),
             metadata=normalized_metadata,
         )
+
+    def to_asset_check_evaluation(
```

Comment:
should this be _to_asset_check_evaluation? or maybe a class method AssetCheckEvaluation.from_asset_check_result() to make it less visible to users

Reply:
As long as we don't give it the `@public` decorator it won't show up in our API docs.

Reply:
thinking about users clicking on it in vscode rather than api docs. Not a big concern